libs/sensors/src/ProximitySwitch.cpp

```diff
@@ -0,0 +1,12 @@
+#include "ProximitySwitch.h"
+
+using namespace Sensor;
+
+ProximitySwitch::ProximitySwitch(PinName proximity_in) : m_proximity_in_adc(proximity_in) {}
+
+/*https://www.dfrobot.com/product-2025.html*/
+float ProximitySwitch::read() {
```

Comment:
You have previously overridden this function, so redefining it will cause an error.